templates/essentials.yaml

```diff
@@ -38,6 +38,10 @@ Parameters:
     AllowedPattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
     ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
     Default: 'it@sagebase.org'
+  MaintenanceWindow:
+    Description: 'Acceptable outage intervals for Systems Manager changes'
+    Type: String
+    Default: 'Weekly-0400SundayUTC-7'
```

Comment:
since there is a default for this param people don't need to set it.  So I'm wondering what is the expectation if people don't set it?  how would we know whether people are really ok with this outage interval?

Reply:
I'm thinking they can choose to opt out if they have a good reason, but it should default to the least invasive maintenance window

Reply:
I think we need more discussion around this tag.  I'm concerned that it can become unclear whether and not usable.  Also this is a bit unrelated to the goal of this PR. 

To unblock this specific PR I would recommend removing this tag and opening a new PR with only this tag change so we can have a PR dedicated to discussing this new tag.